src/Data/Textile/Simulator.elm

```diff
@@ -111,7 +108,8 @@ compute db query =
             else
                 identity
     in
-    init db query
+    Query.handleUpcycling query
+        |> init db
```

Comment:
Just a nit but having the `handleUpcycling` function called here looks strange to me (because everything else is not related to the query). Why not calling the `handleUpcycling` function in the `init` function directly?

Reply:
addressed in https://github.com/MTES-MCT/ecobalyse/pull/710/commits/d81f54b8268b9de23b1260bf01f85bc3bb087734